src/System.IO.IsolatedStorage/ref/System.IO.IsolatedStorage.cs

```diff
@@ -12,7 +12,7 @@ public partial interface INormalizeForIsolatedStorage
         object Normalize();
     }
 
-    public abstract partial class IsolatedStorage // : System.MarshalByRefObject
+    public abstract partial class IsolatedStorage : MarshalByRefObject
```

Comment:
>MarshalByRefObject [](start = 52, length = 18)

I know it's not required, but given this is a ref, might as well just put the full qualified name here.